applications/maplab-server-node/src/maplab-server-ros-node.cc

```diff
@@ -22,35 +22,38 @@
 #include "maplab-server-node/maplab-server-node.h"
 
 DEFINE_string(
-    server_node_config_file, "",
+    maplab_server_node_config_file, "",
     "Path the the config YAML file for the maplab server node.");
 
 DEFINE_int32(
-    map_update_notification_subscriber_queue_size, 100,
-    "Size of ROS subscriber.");
+    maplab_server_map_update_topic_queue_size, 100, "Size of ROS subscriber.");
+
+DEFINE_string(
+    maplab_server_map_update_topic, "map_update_notification",
+    "Topic on which the map update notification message is received, it "
+    "contains the robot name and the map folder of the new map update.");
 
 namespace maplab {
 
-MaplabServerRosNode::MaplabServerRosNode(
-    const MaplabServerRosNodeConfig& config)
-    : config_(config), maplab_spinner_(common::getNumHardwareThreads()) {
+MaplabServerRosNode::MaplabServerRosNode(const MaplabServerNodeConfig& config)
+    : maplab_spinner_(common::getNumHardwareThreads()) {
   LOG(INFO) << "[MaplabServerRosNode] Initializing MaplabServerNode...";
-  maplab_server_node_.reset(new MaplabServerNode(config_.server_config));
+  maplab_server_node_.reset(new MaplabServerNode(config));
 }
 
 MaplabServerRosNode::MaplabServerRosNode(
     const ros::NodeHandle& nh, const ros::NodeHandle& nh_private)
-    : config_(),
-      nh_(nh),
+    : nh_(nh),
       nh_private_(nh_private),
       maplab_spinner_(common::getNumHardwareThreads()) {
-  CHECK(config_.deserializeFromFile(FLAGS_server_node_config_file))
+  MaplabServerNodeConfig config;
+  CHECK(config.deserializeFromFile(FLAGS_maplab_server_node_config_file))
       << "[MaplabServerRosNode] Failed to parse config from '"
-      << FLAGS_server_node_config_file << "'";
+      << FLAGS_maplab_server_node_config_file << "'";
 
   // === MAPLAB SERVER NODE ===
   LOG(INFO) << "[MaplabServerRosNode] Initializing MaplabServerNode...";
-  maplab_server_node_.reset(new MaplabServerNode(config_.server_config));
+  maplab_server_node_.reset(new MaplabServerNode(config));
 
   // Set up map saving service.
```

Comment:
You have repeated twice the same comment here and below. I guess the one below should be map lookup service